eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -38,7 +39,7 @@ steps:
     GdnBreakGdnToolCredScanSeverity: Error
     GdnBreakBaselines: baseline
     # Used for generating baseline file.
-    # GdnBreakOutputBaselineFile: dotnet
+    # GdnBreakOutputBaselineFile: $(Language)
```

Comment:
If language needs to generate new baseline file, use variable in pipeline.

This is intermediate step. Put it in the comments.

Reply:
So this will be eventually uncommented? Or it's supposed to be copied/edited in a language repo?

Reply:
We never check in the uncommented one. The uncommented code will be used privately.

Whenever we want to generate baseline file, we need to uncommented the line, and run pipeline (against PR) to execute the step. Pipeline will publish the newly generated baseline file in `Artifacts`. We can open a PR with new baseline file to its language repo. Then the uncomment change can be abandoned.

Reply:
This still looks like a DevOps variable that I don't think we set anywhere. Instead should we give it a generic name like "baseline" or something that would work for everyone if they need to generate one.